.gitignore

```diff
@@ -2,4 +2,5 @@ build
 coverage
 lib
 node_modules
+.idea/
```

Comment:
I would prefer this to be ignored on the `.gitignore` file of your local machine. We don't want to ignore every IDE generated folders in this project :).

Please remove this line again. 🙌